From 6725c5f0169981cb0bd2ee124fbf0d8ca30b762d Mon Sep 17 00:00:00 2001
From: Steve Lhomme <slhomme@matroska.org>
Date: Sat, 24 Feb 2024 08:33:56 +0100
Subject: [PATCH] drop CMake older than 3.5

It generates a warning with newer versions of CMake.
3.4 was released in 2015. We require C++17 with compilers only available in 2017/2018.
If your CMake is older than your toolchain, you have a problem.

There's no need to support legacy CMake going forward in libebml 2.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 11ee059a..c7b969a8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.1.2)
+cmake_minimum_required(VERSION 3.5)
 
 project(ebml VERSION 2.0.0)
 
